Kernel/API/POSIX/fcntl.h

```diff
@@ -25,10 +25,10 @@ extern "C" {
 
 #define FD_CLOEXEC 1
 
-#define O_RDONLY (1 << 0)
```

Comment:
This means it previously was possible to get an FD that is not even readable by passing the flags as `0` (which is what Zig did to get read-only).

Reply:
🤔 is this one of those "non-portable assumptions that holds everywhere except serenity"? Or is the objectively correct but mostly pointless fix for zig's compiler/runtime/users to be more specific about the posix flags they use?

Reply:
Imagine not linking to IEEE Std 1003.1-2024 in $CURRENT_YEAR https://pubs.opengroup.org/onlinepubs/9799919799/functions/open.html

Reply:
But we are technically within spec either way